WireAPI/Package.swift

```diff
@@ -21,12 +25,19 @@ let package = Package(
         .package(
             url: "https://github.com/pointfreeco/swift-snapshot-testing",
             from: "1.16.0"
+        ),
+        .package(
+            url: "https://github.com/krzysztofzablocki/Sourcery.git",
+            from: "2.2.4"
```

Comment:
This pulls a lot of sub-dependencies in the project that all need to be build. Until now we have used another way with a binary file, @caldrian has some knowledge there.
Can we use it here too?

Reply:
Yeah it'd be nice to pull it once and share it, not sure how though. We could also just not add the dependency to the package and instead generate the mocks via the command line.

Reply:
I removed the plugin for now, we can still generate mocks from the command line by running `sourcery` from the `WireAPI` directory.